project_task_dependency/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Dennis Sluijk <d.sluijk@onestein.nl>
```

Comment:
You can add yourself here, and I also did some optimization changes in v11, so you can add me as well.

Reply:
Done :+1: